packages/x-data-grid-premium/src/DataGridPremium/DataGridPremium.tsx

```diff
@@ -45,22 +44,16 @@ const configuration: GridConfiguration<GridPrivateApiPremium, DataGridPremiumPro
       const apiRef = useGridApiContext();
       return useGridSelector(apiRef, gridCellAggregationResultSelector, { id, field });
     },
-    useSortValueGetter: (apiRef) => (id, field) =>
-      gridCellAggregationResultSelector(apiRef, {
-        id,
-        field,
-      })?.value ?? apiRef.current.getCellValue(id, field),
     useFilterValueGetter: (apiRef, props) => (row, column) => {
```

Comment:
So for the filterValueGetter, we always want to only get row values and no aggregation values, correct?
In that case, this whole function boils down to `apiRef.current.getRowValue` call, and `useFilterValueGetter` can be removed?